pages/linux/aspell.md

```diff
@@ -0,0 +1,23 @@
+# aspell
+
+> Interactive spell checker.
+
+- Spell check a single file:
+
+`aspell check file`
```

Comment:
Could `file` be `{{path/to/file}}` please?

Reply:
done